src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+
+# Actuator health check config
+management.endpoints.web.base-path=/
+management.endpoints.web.path-mapping.health=healthcheck
+
+# Default application root path
+server.servlet.context-path=/item-group-workflow-api
+
+server.port=9090
```

Comment:
Shouldn't be including the server.port when committing the code to the repository

Reply:
good spot, will remove